workflows/Snakefile

```diff
@@ -125,5 +126,72 @@ rule run_mcmc_mark00:
             --out_dir {params.out_dir}
         """
 
```

Comment:
Below: two messy snakemake rules to run the two scripts created. Will rewrite the snakemake completely soon. I'll think about a sensible way to store all data and plots then.